charts/lab-partner/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lab-partner
+  labels:
+    {{- include "lab-partner.labels" . | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
@ChrisSchreiber was this meant to be resolved?  I am still unclear on the reasoning for why we can't have more than one replica.  I think it is worth figuring out how to have an HA deployment so we don't have to worry about downtime during rolling updates.